python/ray/experimental/internal_kv.py

```diff
@@ -21,39 +21,72 @@ def _initialize_internal_kv(gcs_client: GcsClient):
     """Initialize the internal KV for use in other function calls.
     """
     global global_gcs_client, _initialized
+    assert gcs_client is not None
     global_gcs_client = gcs_client
     _initialized = True
 
 
+# b'@:' will be the leading characters for namespace
```

Comment:
Btw why use symbol here?

I think this would be more easily understood if just __NS_START_CHAR = "namespace_". Then the raw keys would look like "namespace_foo:bar"

Reply:
I'm just trying to avoid conflict With the current usage of ray client. Right now ray client can internal kv out and get, so put something special there can avoid the issue.

Reply:
How about: "@namespace" ?